src/audio/Kconfig

```diff
@@ -119,6 +119,17 @@ config COMP_KPB
 	default y
 	help
 	  Select for KPB component
+if COMP_KPB
+
+config KPB_FORCE_COPY_TYPE
```

Comment:
Can we say `KPB_FORCE_COPY_TYPE_X` that way it's easier to follow the options otherise it's ambiguous.

Reply:
Ok, i added KPB_FORCE_COPY_TYPE_NORMAL. In the future we might one to make this a choice with multiple possible values.

Reply:
ok, but what is ```NORMAL`` for KBP is it "one shot", or "split into periods" ? This should be in the naming.

Reply:
@lgirdwood normal is the terminology used by Host. This means 'split into periods'
```
enum comp_copy_type {
»       COMP_COPY_NORMAL = 0,»  /**< Normal */                                                                                                                                                             
»       COMP_COPY_BLOCKING,»    /**< Blocking */
»       COMP_COPY_ONE_SHOT,»    /**< One-shot */
};
```

Reply:
ok, that needs some explaination too then :)